app/models/nic/base.rb

```diff
@@ -271,6 +274,11 @@ def interface_attribute_uniqueness(attr, base = Nic::Base.where(nil))
       db_candidates = db_candidates.select { |c| c.id != self.id && in_memory_candidates.map(&:id).include?(c.id) }
       errors.add(attr, :taken) if db_candidates.present?
     end
+
+    def log_host_orchestration_errors
+      logger.error "Host orchestration errors: #{host.errors.full_messages.to_sentence}" unless orchestration_errors?(host)
```

Comment:
Katello test still fails:
NoMethodError: undefined method `orchestration_errors?' for #<Nic::Base:0x0000001baa0558>